action.yaml

```diff
@@ -104,8 +107,9 @@ runs:
         name: ${{ inputs.supergraph-schema-artifact }}
         path: ./${{ inputs.supergraph-schema-filename }}
 
-    - name: Configure Git
+    - name: Configure Git with App
       shell: bash
+      if: inputs.github-app-id && inputs.github-app-private-key
```

Comment:
Could these just check if `steps.app-token.outputs.token` exists rather than relying on the same `if`? Or maybe `steps.app-token.outcome`?

Reply:
Good call, used the `outcome`